src/resolvers/JsonResolver.ts

```diff
@@ -0,0 +1,23 @@
+import {Resolver} from "./Resolver";
+import {uidModel} from "../resolvers/resolverType";
+
+export class JsonResolver extends Resolver {
+
+    constructor(model:uidModel, name:string, content:any, advancedOptions?:any) {
+       super(model,name,content,advancedOptions);
+    }
+
+    watchDependencies(){}
+
+    resolve() : void {
+        try{
+            Object.defineProperty(this.model,this.name,  {
+                value: JSON.parse(this.content),
+                writable: true
+            });
+        } catch(e){
```

Comment:
Seems a JSON error (e.g. bad syntax) is not catched here, and we never get the error message in the console

Reply:
A test in JsonResolver.spec test if catch is worked